components/managed-identity/inputs-default.tf

```diff
@@ -11,7 +11,7 @@ locals {
   log_analytics_env_mapping = {
     sandbox = ["sbox", "ptlsbox"]
     nonprod = ["dev", "perftest", "ithc", "demo", "aat"]
-    prod    = ["prod", "mgmt"]
+    prod    = ["prod", "mgmt", "ptl"]
```

Comment:
Is this mapping correct?

Reply:
Existing PTL environment is sending to the hmcts-prod log analytics workspace.